src/utils.ts

```diff
@@ -11,7 +11,7 @@ function bytesToHex(a) {
     .join('')
 }
 
-function hexToBytes(a) {
+function hexToBytes(a): number[] {
```

Comment:
why not Uint8Array? 

here is a tested implementation that is compatible in node and in browser: https://github.com/xpring-eng/xpring-common-js/blob/master/src/utils.ts#L134

Reply:
I tried to make this change but it necessitates other changes throughout the code. Uint8Array is new and I would be fine with using it, but it's going to require a larger effort that can be done in a future PR.